datafusion/core/tests/dataframe.rs

```diff
@@ -245,7 +248,7 @@ async fn test_count_wildcard_on_where_scalar_subquery() -> Result<()> {
                     .await?
                     .filter(out_ref_col(DataType::UInt32, "t1.a").eq(col("t2.a")))?
                     .aggregate(vec![], vec![count(lit(COUNT_STAR_EXPANSION))])?
-                    .select(vec![count(lit(COUNT_STAR_EXPANSION))])?
+                    .select(vec![col("COUNT(*)")])?
```

Comment:
Since this is a dataframe test, maybe could use the expr name directly rather than the hard coded string 🤔 

Something like

```rust
               let count_star = count(lit(COUNT_STAR_EXPANSION));
                ctx.table("t2")
                    .await?
                    .filter(out_ref_col(DataType::UInt32, "t1.a").eq(col("t2.a")))?
                    .aggregate(vec![], vec![count_star.clone()])?
                    .select(vec![col(count_star.to_string())])?
```

Reply:
A great idea!